src/test/java/racingcar/race/CarTest.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.race;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+
+    @DisplayName("drive 를 통해 position을 변경하면 정상적으로 변경되는 지 확인")
+    @Test
+    void drive() {
+        Car car = new Car();
+        car.drive(5);
+        int firstDrivePosition = car.getPosition();
+        car.drive(3);
+        int secDrivePosition = car.getPosition();
+
+        assertThat(firstDrivePosition).isEqualTo(5);
+        assertThat(secDrivePosition).isEqualTo(8);
```

Comment:
한 Car에 대한 position 체크를 해보고 싶으시다면, 
```suggestion
    void drive() {
        Car car = new Car();
        car.drive(5);
        int firstDrivePosition = car.getPosition();
        assertThat(firstDrivePosition).isEqualTo(5);
        
        car.drive(3);
        int secDrivePosition = car.getPosition();
        assertThat(secDrivePosition).isEqualTo(8);
```

이렇게 사용하는 건 어떨까요?

Reply:
괜찮은 것 같아요. 수정하겠습니다.